config/environments/test.rb

```diff
@@ -52,6 +52,7 @@
   config.x.otp_secret = '100c7faeef00caa29242f6b04156742bf76065771fd4117990c4282b8748ff3d99f8fdae97c982ab5bd2e6756a159121377cce4421f4a8ecd2d67bd7749a3fb4'
 
   # Generate random VAPID keys
+  # Uses Webpush class, Webpush::Legacy does the same thing.
```

Comment:
I think this comment can be removed, `Webpush::Legacy` (as in, in the gem maintained by Mastodon) does not cover this at all.